mypy/main.py

```diff
@@ -164,7 +164,7 @@ def _fill_text(self, text: str, width: int, indent: str) -> str:
     ('allow', 'disallow'),
     ('show', 'hide'),
 ]  # type: Final
-flag_prefix_map = {}  # type: Final[Dict[str, str]]
+flag_prefix_map = {}  # type: Final
```

Comment:
I'd prefer to leave this as is, since the type is not immediately obvious. The type comments acts as documentation.